RELEASE.md

```diff
@@ -1,20 +1,17 @@
 # Cutting a Jaeger UI release
 
-1. Determine the version for the release.
-   - Follow [semver.org](https://semver.org) to determine the new version for Jaeger UI.
-     - Review all changes since the last release to determine how, if at all, any externally facing APIs are impacted. This includes, but is not limited to, the UI config and URL routes such as deep-linking and configuring the embedded mode.
-   - Preface the version with a "v", e.g. `v1.0.0`.
-1. Create and merge, per approval, a PR which preps the release.
+1. Create and merge, per approval, a PR which preps the release ([example](https://github.com/jaegertracing/jaeger-ui/pull/1767)).
    1. The PR title should match the format "Prepare release vX.Y.Z".
-   1. CHANGELOG.md
-      - Change the version of the current release from "Next (unreleased)" to "vX.Y.Z (Month D, YYYY)" where "vX.Y.Z" is the semver for this release.
-      - Make sure all relevant changes made since the last release are present and listed under the current release.
-        - Tip: Run `node scripts/get-changelog.js` script that prints a dozen most recent commits in the chagelog format.
+   2. CHANGELOG.md
+      - Change the version of the current release from "Next (unreleased)" to "vX.Y.Z (Month D, YYYY)" where "vX.Y.Z" is the [semver](https://semver.org) for this release.
+      - Run `make changelog` to list all changes since the last release.
```

Comment:
Given the jaeger maintainers are (currently) releasing both jaeger-ui and jaeger, I thought to make the experience more consistent between the two repos, hence the introduction of a Makefile and the same python scripts used to generate the changelog. Please let me know if you disagree and I can revert.

If we agree to this change, I could also remove the existing `get-changelog.js` and `get-tracking-version.js` scripts  _if_ they're not used for any other purpose other than for releases.